### PR TITLE
Added specs for NB-Controls:

### PR DESCRIPTION
**Pull request by MSFT-Chirag**
Added specs for NB-Controls:

1. Azure_DBforPostgreSQL_NetSec_Dont_Allow_Universal_IP_Range
2. Azure_DBforMySQL_AuthZ_Firewall_Deny_AzureServices_Access
3. Azure_RedisCache_BCDR_Use_RDB_Backup

### CODE REVIEW

**Review by gvaradarajan-msft — APPROVED** [review]

File: Control coverage/Feature/DBForMySql.md [hunk 1]
```diff
@@ -15,6 +16,61 @@
...
+- **Enforcement Policy** 
+
+	 [![Link to Azure Policy](https://raw.githubusercontent.com/MSFT-Chirag/AzTS-docs/main/Assets/View_Definition.jpg)](https://portal.azure.com/#blade/Microsoft_Azure_Policy/CreatePolicyDefinitionBlade/uri/<policy-raw-link>) 
+
+	 [![Link to Azure Policy](https://raw.githubusercontent.com/MSFT-Chirag/AzTS-docs/main/Assets/Deploy_To_Azure.jpg)](https://portal.azure.com/#blade/Microsoft_Azure_Policy/CreatePolicyDefinitionBlade/uri/<policy-raw-link>)  -->
+
+### Azure Policy or ARM API used for evaluation 
+
+- ARM API to get firewall rules of a DBForMySql server: /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DBforMySQL/servers/{serverName}/firewallRules/{firewallRuleName}?api-version=2017-12-01 <br />
+**Properties:** name
```
Comment (VishalHaibatpure) [inline Control coverage/Feature/DBForMySql.md:67]: This should list properties used for valuation.  Please check what properties are used below
Turn 'OFF' the 'Allow access to Azure services' setting.
Remove IP range from firewall rules.

File: Control coverage/Feature/DBForPostgreSQL.md [hunk 2]
```diff
@@ -119,6 +120,61 @@ Using the firewall feature ensures that access to the data or the service is res
...
+
+	 [![Link to Azure Policy](https://raw.githubusercontent.com/MSFT-Chirag/AzTS-docs/main/Assets/View_Definition.jpg)](https://portal.azure.com/#blade/Microsoft_Azure_Policy/CreatePolicyDefinitionBlade/uri/<policy-raw-link>) 
+
+	 [![Link to Azure Policy](https://raw.githubusercontent.com/MSFT-Chirag/AzTS-docs/main/Assets/Deploy_To_Azure.jpg)](https://portal.azure.com/#blade/Microsoft_Azure_Policy/CreatePolicyDefinitionBlade/uri/<policy-raw-link>)  -->
+
+### Azure Policy or ARM API used for evaluation 
+
+- ARM API to fetch the firewall rules for a DBForPostgreSQL server: 
+/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DBforPostgreSQL/servers/{serverName}/firewallRules?api-version=2017-12-01 <br />
+**Properties:** name
```
Comment (VishalHaibatpure) [inline Control coverage/Feature/DBForPostgreSQL.md:171]: Same as above

File: Control coverage/Feature/DBForMySql.md (hunk 1 above)
Comment (MSFT-Chirag) [reply Control coverage/Feature/DBForMySql.md:67]: 'Name' is the only property used for evaluation.

File: Control coverage/Feature/DBForPostgreSQL.md (hunk 2 above)
Comment (MSFT-Chirag) [reply Control coverage/Feature/DBForPostgreSQL.md:171]: Name is the property used for evaluation.

**Review by TarunKrShukla — APPROVED** [review]
Looks good

**Pull request merged**